packages/ui/src/primitives/tailwind/Modal/index.tsx

```diff
@@ -87,20 +103,23 @@ export const ModalFooter = ({
   closeButtonText?: string
   submitButtonText?: string
   showCloseButton?: boolean
+  startSubmitIcon?: ReactNode
 }) => {
   const { t } = useTranslation()
   return (
     <div className="grid grid-flow-col border-t border-t-theme-primary px-6 py-5">
       {showCloseButton && (
-        <Button variant="outline" disabled={closeButtonDisabled} onClick={() => onCancel && onCancel(false)}>
+        <Button variant="secondary" disabled={closeButtonDisabled} onClick={() => onCancel && onCancel(false)}>
```

Comment:
yes maybe we need to rethink this because right now the ThemeService works for every project that use the talwind components, we need to create a theme for each project

Reply:
yes we can extend the tailwind config in that case